From 25d2ae74941390e9fd65d4aec1e4c0648c7af314 Mon Sep 17 00:00:00 2001
From: Ioan Sucan <isucan@gmail.com>
Date: Sat, 15 Feb 2014 20:56:12 -0800
Subject: [PATCH] fix install rules

---
 CMakeLists.txt | 12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 33a7d34..4d26ec4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -128,8 +128,14 @@ install(DIRECTORY include/ DESTINATION include
   PATTERN ".DS_Store" EXCLUDE
 )
 
-install(FILES "${pkg_conf_file}" DESTINATION lib/pkgconfig/ COMPONENT pkgconfig)
+find_package(catkin)
+catkin_package(
+  INCLUDE_DIRS include
+  LIBRARIES ${PROJECT_NAME}
+  )
 
+#install(FILES "${pkg_conf_file}" DESTINATION lib/pkgconfig/ COMPONENT pkgconfig)
 
-enable_testing()
-add_subdirectory(test)
+
+#enable_testing()
+#add_subdirectory(test)
-- 
2.1.0.rc2.206.gedb03e5

